RecoLocalCalo/HcalRecProducers/python/facileHcalReconstructor_cfi.py

```diff
@@ -0,0 +1,17 @@
+import FWCore.ParameterSet.Config as cms
+
+sonic_hbheprereco = cms.EDProducer("FacileHcalReconstructor",
+    Client = cms.PSet(
+        batchSize = cms.untracked.uint32(16000),
+        address = cms.untracked.string("0.0.0.0"),
+        port = cms.untracked.uint32(8001),
+        timeout = cms.untracked.uint32(300),
+        modelName = cms.string("facile_all_v5"),
+        mode = cms.string("Async"),
+        modelVersion = cms.string(""),
+        verbose = cms.untracked.bool(False),
+        allowedTries = cms.untracked.uint32(5),
```

Comment:
it would be good to justify the choice of allowed failures here.

Reply:
We haven't observed any failures in any test to date. We set this to 0 for now.